.ebextensions/deploy.config

```diff
@@ -0,0 +1,118 @@
+option_settings:
+  - namespace: aws:autoscaling:launchconfiguration
+    option_name: BlockDeviceMappings
+    value: /dev/sdh=:10
+  - option_name: BUNDLE_DISABLE_SHARED_GEMS
+    value: "1"
+  - option_name: BUNDLE_PATH
+    value: "vendor/bundle"
+
+packages:
+  yum:
+    git: []
+
+files:
+  "/etc/nginx/conf.d/proxy.conf" :
+    mode: "000755"
```

Comment:
For all of these files, you use permission mode 755 (-xwrx-rx-r). Is this the appropriate permission mode? Since the owning user and group are both root (and I'd assume the only user in group root is user root), root and group permissions should be the same, in theory. Also, who will be running these scripts? Write access only going to root makes sense, but should all users be able to execute these files?